### ISSUE TITLE
create a good looking landing page for opencode

### ISSUE THREAD

**Issue by Anupam-dagar** [opencode18/ProDesigner#95]
take inspiration from current landing page of opencode18 website.

**Comment by ekanban** [comment]
is this for everyone?

**Comment by Anupam-dagar** [comment]
yes

**Comment by ananya** [comment]
https://ananya-agrawal.github.io/opencode18.github.io/
@Anupam-dagar

**Comment by gauthamzz** [comment]
Need codepen

**Comment by ananya** [comment]
@gauthamzz as the landing page animation was meant to be done on opencode website so i did that on the website itself and not only on a part
How can i make a codepen of this, this is a complete website?

**Comment by gauthamzz** [comment]
Well we only take codepens in this repo and I don’t prefer using a design already used in girls who code

**Comment by ananya** [comment]
@gauthamzz how could i make a codepen of this please tell.
and sorry but i asked anupam sir before adding the same animation and he said he is okay with it .

**Comment by gauthamzz** [comment]
Put HTML , css and js in the place how you normally make codepen

**Comment by gauthamzz** [comment]
If he said it’s fine, I’ll
Merge but only codepen

**Comment by Anupam-dagar** [comment]
@ananya-agrawal I think there was a misunderstanding in it.I agreed for the animation but not for implementing on website, this repo is supposed to take codepen links only
Also a landing page is needed, so the rest of the contents are redundant for this issue.
https://codepen.io/marcobiedermann/pen/mevPWe?q=landing+page&limit=all&type=type-pens
The above link is a perfect example for building landing pages on codepen.

**Comment by ananya** [comment]
@Anupam-dagar yes sir it was me only who thought that issue is to be solved on original website only

**Comment by Anupam-dagar** [comment]
@ananya-agrawal no sir system :stuck_out_tongue: 
No problem, it happens.
